src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/PlatformManager.java

```diff
@@ -52,12 +69,13 @@ public PlatformManager(List<Platform> crowdPlatforms, WorkerIdentification fallb
 
         //create hashmap of platforms
         platforms = crowdPlatforms.stream()
-                .collect(Collectors.toMap(Platform::getName, Function.identity()));
+                .collect(Collectors.toMap(Platform::getType, Function.identity()));
```

Comment:
It must be possible that the identification differs for two platforms of the same type. Could also be two totallly unrelated Pybossa instances.